packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -118,66 +122,29 @@ export const Product = ({
     selectedPlan = productPlans[0];
   }
 
-  const onPayment = useCallback(
```

Comment:
Most of the code removal here in the Product route is stuff moved to SubscriptionCreate